### PR TITLE
chore: add short bio for ross

### PR DESCRIPTION
**Pull request by rc-swag**
Add short bio in the about developer team for Ross Cruickshank.

Fixes https://github.com/keymanapp/keyman.com/issues/435

### CODE REVIEW

File: about/developers/index.md [hunk 1]
```diff
@@ -100,6 +100,10 @@ I am married, with two adult children, and two cats.  I love playing computer ga
 
 My personal website can be found at [markcsinclair.co.uk](https://markcsinclair.co.uk/)
 
+### Ross Cruickshank
+<br/>
+With degrees in Electronics and Information Technology from QUT, I have worked across various roles in software and electronics engineering. This has given me experience in high-availability systems, microcontroller design, and software development. With Keyman, I contribute mostly to the Keyman for Windows product. I am based in Australia and outside of work, I enjoy spending time with my family, mountain biking and road cycling.
+
```
Comment (darcywong00) [inline about/developers/index.md:106]: Can add the GH avatar

```suggestion

----

![](https://github.com/rc-swag.png?size=240)
```

**Review by darcywong00 — APPROVED** [review]
lgtm

File: about/developers/index.md (hunk 1 above)
Comment (darcywong00) [reply about/developers/index.md:106]: Fixes #435

Comment (rc-swag) [reply about/developers/index.md:106]: good suggestion about the avatar I will make the change

**Pull request merged**